lib/credit_card_sanitizer.rb

```diff
@@ -131,19 +132,4 @@ def without_expiration(text)
     yield
     text.gsub!(expiration_date_boundary, '')
   end
-
-  if ''.respond_to?(:scrub)
-    def to_utf8!(str)
-      str.force_encoding(Encoding::UTF_8)
```

Comment:
yes, just ran into the same in classic ... does not change the encoding ... and if the encoding is binary then it is a noop ...